.github/workflows/check-full.yaml

```diff
@@ -12,31 +12,44 @@ jobs:
   R-CMD-check:
     runs-on: ${{ matrix.config.os }}
 
-    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }}) ${{ matrix.config.locale }}
 
     strategy:
       fail-fast: false
       matrix:
         config:
-          - {os: macos-latest,   r: 'release'}
+          - { os: macos-latest, r: "release" }
 
-          - {os: windows-latest, r: 'devel'}
-          - {os: windows-latest, r: 'release'}
+          - { os: windows-latest, r: "devel" }
+          - { os: windows-latest, r: "release" }
```

Comment:
The formatting changes are coming from Prettier formatter in VS code. They are all desirable.

Reply:
Sure.